instagram-app/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Comment from "./Comment";
+import CommentInput from "./CommentInput";
+
+class CommentSection extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comments: props.comments
+    };
+  }
+
+  render() {
+    return (
+      <div>
+        {this.state.comments.map((c, i) => (
+          <Comment key={i} comment={c} />
+        ))}
+        <CommentInput />
+      </div>
+    );
+  }
+}
+
+CommentSection.propTypes = {
+  comments: PropTypes.arrayOf(
+    PropTypes.shape({
+      text: PropTypes.string,
+      username: PropTypes.string
```

Comment:
When I type a comment, there appears a warning that indicates *comment.username* is marked as required but its value is `null`.